third_party/npm_@trezor_connect/LICENSE

```diff
@@ -1,3 +1,11 @@
+Brave has received this extra permission from the Trezor CTO:
```

Comment:
It would probably be good to mention the CTO's name and date if doing this anyway.
Oct 12, 2021 Vojtech Cerny
Or at least his first name

Reply:
Good idea. Added.